index.html

```diff
@@ -0,0 +1,217 @@
+                <div class="icon-container">
+                <img src="./images/lecture.png" alt="Lecture icon">
+                </div>
+                <h3 class="lecture-title">Lecture</h3>
+                <p class="lecture-desc">We have curated and lined up excellent keynotes from industry leaders.</p>
+
+            </div>
+            <div class="lecture">
+                <div class="icon-container">
+                <img src="./images/CC.png" alt="Exhibition icon">
+                </div>
+                <h3 class="lecture-title">Exhibition</h3>
+                <p class="lecture-desc">innovations from emerging tech hubs across Africa will be showcased.</p>
+            </div>
+            <div class="lecture">
+                <div class="icon-container">
+                <img src="./images/forum.png" alt="Forum icon">
+                </div>
+                <h3 class="lecture-title">forum</h3>
+                <p class="lecture-desc">Forum to engage industry leaders with time to share your opinions.</p>
+            </div>
+            <div class="lecture">
+                <div class="icon-container">
+                <img src="./images/workshop.png" alt="Workshop icon">
+                </div>
+                <h3 class="lecture-title">Workshop</h3>
+                <p class="lecture-desc">Rather than observation, there will be flexibility for hands on learning</p>
+            </div>
+            <div class="lecture">
+                <div class="icon-container">
+                <img src="./images/ignite.png" alt="Ignite icon">
+                </div>
+                <h3 class="lecture-title">Ignite</h3>
+                <p class="lecture-desc">Network with IT leaders from all around Africa and the world.</p>
+            </div>
+        </div>
+        <div class="btn-container">
+            <button class="btn">Join The Africa Tech Summit 2022</button>
+            </div>
+
+    </section>
+
+    <section class="speakers">
+        <div class="featured">
+            <h3>Featured Speaker</h3>
+            <hr>
+        </div>
+        <div class="card-container">
+        <!-- <div class="card">
+            <img src="./images/first.png" alt="Deborah Ojengbede">
+            <div class="bio">
+            <h3>Deborah Ojengbede</h3>
+            <p id="title"><em>Deborah Ojengbede is the CEO of AFEN Blockchain Group</em></p>
+            <hr>
+            <p>A seasoned banker with over 5 years of working experience in the Banking industry.</p>
+            </div>
+        </div>
+        <div class="card">
+            <img src="./images/second.png" alt="Emmanuel Lubanzadio">
+            <div class="bio">
+            <h3>Emmanuel Lubanzadio</h3>
+            <p id="title"><em>Emmanuel Lubanzadio is Head of Public Policy for Sub-Saharan Africa on Twitter’s Public Policy EMEA Team in Dublin.</em></p>
+            <hr>
+            <p>A seasoned business analyst with over 10 years of working experience in the Tech industry</p>
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.

Reply:
Changes made